### DISCUSSION TITLE
Quick TX lock or simpler TX unlocking

### DISCUSSION THREAD

**Discussion by qsus** [AubsUK/uv-k5-firmware-custom#16] [Ideas]
Hi,
I don't usually TX, so I want to lock TX in order not to transmit unintentionally, but when I do want to TX, it takes too much effort to change TX settings in the hidden menu, especially when changing to unlock all.
Could you please remove having to enable unlock all 10 times, at least as a build option? And maybe unhide the hidden menu as a build option? Or add some kind of TX lock, which could by quickly enabled using the side buttons maybe?
Thanks

**Comment by yosemitez** [comment]
i am going to request a menu>keylock option as found on 4fhwn that sets keylock to keylock + lock ptt. it is brillant as i just use my uvk5 as a scanner.

with f4hwn when i turn on the radio the ptt is locked and it starts scanning the last selected bank!
